README.md

```diff
@@ -48,6 +48,8 @@ Any Windows instances that are to be attached to the cluster as a node must fulf
 A ConfigMap named `windows-instances` must be created in the WMCO namespace, describing the instances that should be
 joined to a cluster. The required information to configure an instance is:
 * An address to SSH into the instance with. This can be a DNS name or an ipv4 address.
+    * A PTR record must exist corresponding to the address used in the configMap resolving to the instance host name for
```

Comment:
configMap -> ConfigMap
host name -> hostname

Reply:
This doesnt seem like it should be a sub-bullet. This is separate from having address to SSH into.
Shouldnt this go into the BYOH instance pre-req doc?

Reply:
Good point, @sebsoto. +1